packages/core/template_project/app/src/main/AndroidManifest.xml

```diff
@@ -61,6 +76,20 @@
                 android:value="<%= metadata.value %>" />
         <% } %>
 
+        <% if (isMetaQuest) { %>
+        <meta-data
+            android:name="com.oculus.pwa.NAME"
+            android:value="@string/appName" />
+
+        <meta-data
+            android:name="com.oculus.pwa.START_URL"
+            android:value="@string/launchUrl" />
+
+        <meta-data
+            android:name="com.oculus.pwa.SCOPE"
+            android:value="@string/launchUrl" />
```

Comment:
This is a temporary stub. A (new?) string representing the actual absolute scope is needed.